src/main/java/de/metas/ui/web/quickinput/invoiceline/InvoiceLineQuickInputProcessor.java

```diff
@@ -45,6 +53,22 @@ public DocumentId process(final QuickInput quickInput)
 		final I_C_Invoice invoice = quickInput.getRootDocumentAs(I_C_Invoice.class);
 		final IInvoiceLineQuickInput invoiceLineQuickInput = quickInput.getQuickInputDocumentAs(IInvoiceLineQuickInput.class);
 
+		final I_C_BPartner partner = invoice.getC_BPartner();
```

Comment:
Don't load the BP here. Load it only if it's needed (i.e. for error message).

Reply:
done